MediaManager/Media/IMediaExtractor.cs

```diff
@@ -11,6 +12,8 @@ public interface IMediaExtractor
 
         Task<IMediaItem> CreateMediaItem(string url);
 
+        Task<IMediaItem> CreateMediaItem(string resourceName, Assembly assembly);
```

Comment:
Assembly assembly = null

Reply:
Added `TryFindAssembly` in `MediaExtractorBase`